modules/navier_stokes/test/tests/finite_volume/cns/straight_channel_porosity_step/hllc.i

```diff
@@ -0,0 +1,303 @@
+p_initial=1.01e5
+T=273.15
+# u refers to the superficial velocity
+u_in=1
+
+[GlobalParams]
+  fp = fp
+  two_term_boundary_expansion = true
+[]
+
+[Mesh]
+  [cartesian]
+    type = GeneratedMeshGenerator
+    dim = 1
+    xmin = 0
+    xmax = 18
+    nx = 180
+  []
+  [to_pt5]
+    input = cartesian
+    type = SubdomainBoundingBoxGenerator
+    bottom_left = '2 0 0'
+    top_right = '4 1 0'
+    block_id = 1
+  []
+  [pt5]
+    input = to_pt5
+    type = SubdomainBoundingBoxGenerator
+    bottom_left = '4 0 0'
+    top_right = '6 1 0'
+    block_id = 2
+  []
+  [to_pt25]
+    input = pt5
+    type = SubdomainBoundingBoxGenerator
+    bottom_left = '6 0 0'
+    top_right = '8 1 0'
+    block_id = 3
+  []
+  [pt25]
+    input = to_pt25
+    type = SubdomainBoundingBoxGenerator
+    bottom_left = '8 0 0'
+    top_right = '10 1 0'
+    block_id = 4
+  []
+  [to_pt5_again]
+    input = pt25
+    type = SubdomainBoundingBoxGenerator
+    bottom_left = '10 0 0'
+    top_right = '12 1 0'
+    block_id = 5
+  []
+  [pt5_again]
+    input = to_pt5_again
+    type = SubdomainBoundingBoxGenerator
+    bottom_left = '12 0 0'
+    top_right = '14 1 0'
+    block_id = 6
+  []
+  [to_one]
+    input = pt5_again
+    type = SubdomainBoundingBoxGenerator
+    bottom_left = '14 0 0'
+    top_right = '16 1 0'
+    block_id = 7
+  []
+  [one]
+    input = to_one
+    type = SubdomainBoundingBoxGenerator
+    bottom_left = '16 0 0'
+    top_right = '18 1 0'
+    block_id = 8
+  []
+[]
+
+[Modules]
+  [FluidProperties]
+    [fp]
+      type = IdealGasFluidProperties
+    []
+  []
+[]
+
+[Problem]
+  kernel_coverage_check = false
```

Comment:
```suggestion
```

Reply:
I really wish that jump to conversation worked properly when github decides to not have every diff loaded. I have to have two windows open to know which file to navigate to in order to add the suggestion to the batch